test/test_core.py

```diff
@@ -412,8 +412,7 @@ def assertStartswith(self, output, prefix):
   def verify_in_strict_mode(self, filename):
     js = read_file(filename)
     filename += '.strict.js'
-    with open(filename, 'w') as outfile:
```

Comment:
so is it just the case now that we always have to specify the encoding explicitly every time we open a file?

Reply:
It differs per-platform.  On windows the default is the system encoding which is normally *not* urt-8.  On linux and mac the default is always utf-8 until you really really ask for it not to be.

This means that throughout emscripten we always need to force `utf-8` or we risk breaking folks (mostly windows folks).  

Likely we have some tests now that will allow us to reproduce the same failures on non-windows.